app/assets/stylesheets/base/govuk/components/_inset-text.scss

```diff
@@ -1,11 +1,30 @@
 .govuk-inset-text {
-  .govuk-summary-list__value & {
+  .govuk-summary-list__value &, .link-list & {
```

Comment:
im not sure i see why you need to create this custom class `.link-list` ?

Reply:
There is another component .link-list which I've added in a previous PR - it is in fact quite similar to summary-list, but it is used for displaying links to the schools below the school group page. Without that change, schools with incomplete profiles would have quite a lot of margin around them within that list.

Reply:
yeah i checked the review app and it seems just spacing styles, so can they be applied with govuk classes and then we dont need `link-list` ?

Reply:
`link-list` adds spacing and borders, it has some special treatment for the first child as well and different spacing rules for headings. However, it is possible that we could just use summary-list for it - I remember I've tried that and had some spacing issues, but I might be able to resolve them. I'll have a go then :)

Reply:
its nice to use govuk classes and styles if possible. let me know if u need a hand https://design-system.service.gov.uk/